ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerVnextDemoController.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+import SwiftUI
+
+// MARK: DrawerContentController
+
+class DrawerContentController: DemoController {
+
+    public func actionViews() -> [UIView] {
+        let spacer = UIView()
+        spacer.backgroundColor = .orange
+        spacer.layer.borderWidth = 1
+        spacer.heightAnchor.constraint(greaterThanOrEqualToConstant: 20).isActive = true
+
+        var views = [UIView]()
+        views.append(createButton(title: "Dismiss", action: { [weak self ] _ in
+            if let strongSelf = self {
+                strongSelf.dismissButtonTapped()
+            }
+        }).view)
+        views.append(createButton(title: "Dismiss (no animation)", action: { [weak self ] _ in
+            if let strongSelf = self {
+                strongSelf.dismissNotAnimatedButtonTapped()
+            }
+        }).view)
+        views.append(spacer)
+        return views
+    }
+
+    public func containerForActionViews() -> UIView {
+        let container = DemoController.createVerticalContainer()
+        for view in actionViews() {
+            container.addArrangedSubview(view)
+        }
+        container.addBackground(color: Colors.surfacePrimary)
```

Comment:
This is demo code not `FluentUI`. Since we are sharing components like images, Democontroller with rest of the Demo code the Colors library is part of it. Adding `StyleSheetManager` to demo code would give an indication that it's ok for client to use the internal style or theme sheet for their own content. 

We don't want client to use `StyleSheetManager` directly. At some point we will replace `Color` wrapper to use `Stylesheet` and clients will continue using their current API. At least that is what I think is the expected functionality.

Reply:
The client app will use StylesheetManager directly to override Themes in our current design. That is where it can access all the appearance proxies (including colors).

Although it's not a big deal to have these calls here now, it'll be more work when we discontinue the Colors class.